custom-metrics-stackdriver-adapter/examples/direct-to-sd/README.md

```diff
@@ -0,0 +1,151 @@
+# Custom Metrics with Stackdriver
+
+This is a minimal example showing how to export custom metrics directly to Stackdriver
+in a k8s cluster running on GCE or GKE. This example assumes that you already have
+a [custom metrics setup] in your cluster.
+
+## Stackdriver dummy exporter
+
+A simple sd-dummy-exporter container exports a metric of constant value 
+to Stackdriver in a loop. The metric name and value can be passed in via flags.
+Pod id is passed to the container via downward API (see [custom-metrics-sd deployment]
+for how it's done)
+
+## Horizontal Pod Autoscaling object
+
+In the example, there is a [Horizontal Pod Autoscaler object] configured to scale the deployment on 
```

Comment:
Nit: please remove the repetition ("configured").
E.g. "In the example, a [Horizontal Pod Autoscaler object] is configured to scale the deployment on the exported metric with a target value lower than the default metric value."

Reply:
Done